tools/proofers/previewControl.js

```diff
@@ -168,12 +168,9 @@ $( function() {
         // set up the font selector
         Object.keys(fontStyles).forEach(function(index) {
             let fontStyle = fontStyles[index];
-            if(fontStyle) {
-                // do not add empty string
-                let selected = (index === previewStyles.defFontIndex);
-                let option = new Option(fontStyle, index, selected, selected);
-                $fontSelector[0].add(option, null);
-            }
+            let selected = (index === previewStyles.defFontIndex);
```

Comment:
This doesn't belong in this MR since it's unrelated to the image control.

Reply:
Unfortunately this branch changed the get_font_styles() function to remove empty strings which seemed like a good idea at the time since it avoids the function building the selector having to check for empty strings but it meant that preview control.js also had to be changed.

Reply:
Unfortunately this branch changed the get_font_styles() function to remove empty strings which seemed like a good idea at the time since it avoids the function building the selector having to check for empty strings but it meant that preview control.js also had to be changed.